### ISSUE TITLE
As a screen reader user, I should be able to navigate and use the site with no difficulty

### ISSUE THREAD

**Issue by ericahagle** [ericahagle/travel-tracker#41]

**Comment by ericahagle** [comment]
Screen readers are tricky if you don't know how to use them, but you can access (and read) all elements using the correct keystrokes.
